pkg/api/v1beta3/dynakube/kspm.go

```diff
@@ -0,0 +1,57 @@
+package dynakube
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type KspmSpec struct {
```

Comment:
what is Kspm? :)

Reply:
is it [Kubernetes Security Posture Management](https://www.sentinelone.com/blog/what-is-kspm/) ?

Reply:
Yes, I would assume so as well 👍 
I will add a comment